README.md

```diff
@@ -5,66 +5,28 @@
 
 [![CircleCI](https://circleci.com/gh/uavaustin/target-finder-model/tree/master.svg?style=svg)](https://circleci.com/gh/uavaustin/target-finder-model/tree/master)
 
-*Models are released in tar archives with Github Releases. Model versions do
-not necessarily correspond to a target-finder version.*
 
-## Building the Model
+## Developer Instructions
 
-The build process for the model is broken up into 4 steps:
+### Install
+1. Download `git clone https://github.com/uavaustin/target-finder-model`
+2. Get Darknet (requires Linux or [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10))
+    * `cd target-finder-model && git clone https://github.com/AlexeyAB/darknet.git`
```

Comment:
Can you add darknet as submodule to this repo.

Reply:
Would probably requiring changing the CI a little as well. Is better since the version will be set to prevent unexpected failures from updates, but still can be updated as needed..

Reply:
Done.

Reply:
readme still indicates to clone it instead of fetch the submodule, after this the PR is good to me.